README.md

```diff
@@ -131,37 +131,39 @@ The following steps describe how to create the service principal, assign the rol
 1. Open the Azure Cloud Shell at [https://shell.azure.com](https://shell.azure.com). You can alternately use the [Azure CLI](https://docs.microsoft.com/cli/azure/install-azure-cli?view=azure-cli-latest) if you've installed it locally. (For more information on Cloud Shell, see the [Cloud Shell Overview](https://docs.microsoft.com/azure/cloud-shell/overview).)
 
     1.1 **(Required ONLY when environment is Azure Stack Hub)** Run the following command to set the SQL Management endpoint to 'not supported'
-      ```bash  
+      ```bash
 
-      az cloud update -n {environmentName} --endpoint-sql-management https://notsupported 
+      az cloud update -n {environmentName} --endpoint-sql-management https://notsupported
 
       ```
-  
+
 2. Use the [az ad sp create-for-rbac](https://docs.microsoft.com/cli/azure/ad/sp?view=azure-cli-latest#az_ad_sp_create_for_rbac) command to create a service principal and assign a Contributor role:
 
+    For web apps (also more secure)
+
     ```azurecli
     az ad sp create-for-rbac --name "{sp-name}" --sdk-auth --role contributor \
         --scopes /subscriptions/{subscription-id}/resourceGroups/{resource-group}/providers/Microsoft.Web/sites/{app-name}
     ```
 
+    For other usage
```

Comment:
can you change the wording here instead of "other usage" ?

Reply:
Sure, I have some alternatives. Something more specific:

> For usage with other Azure services (Storage Accounts, Active Directory, etc.)

Or more general like:

> For non-web app usage

Reply:
please use this:
For usage with other Azure services (Storage Accounts, Active Directory, etc.)